### DISCUSSION TITLE
Is it possible to change the tagging scheme for NER training?

### DISCUSSION THREAD

**Discussion by fatemerhmi** [explosion/spaCy#9870] [Help: Model Advice] [answered]
Hello, 
I am under the impression that the available models are being trained with BIO scheme for Named Entity Recognition task. If I am training a new model with a config file (using spaCy v3.2). Is it possible to configure which scheme I want the model train and evaluate on? (i.e IO or BILOU)
Also is it possible to see a model evaluation for each tag? Like what would be the precision and recall for B-GPE, I-GPE and O separately. 

Thanks

**Comment by adrianeboyd** [comment, marked as answer]
Underneath the models are trained on BILUO tags and it's not possible to change this. The entity tags predicted by the model are saved on the doc's tokens in IOB format (technically it's IOB2 format with starting `B` tags for all spans), so that's what you see in `token.ent_iob_` and `token.ent_type_`.

You can see the evaluation for each type (`GPE`, `PERSON`, etc.) in the `ents_per_type` scores. If you run `spacy evaluate`, you should see this as a table with PRF scores for each entity type following the overall `ENTS_P/R/F` scores. In the NER evaluation, any entity span that's not 100% correct (correct start token, end token, and entity label) is counted as incorrect.
